pkg/ui/distccl/BUILD.bazel

```diff
@@ -0,0 +1,17 @@
+load("@io_bazel_rules_go//go:def.bzl", "go_library")
+
+genrule(
+    name = "assets",
+    srcs = ["//pkg/ui/workspaces/db-console:db_console_webpack"],
+    outs = ["assets"],
```

Comment:
So generally you [should](https://docs.bazel.build/versions/main/be/general.html#general-advice) avoid creating directories as `genrule` outputs:

> Avoid creating symlinks and directories. Bazel doesn't copy over the directory/symlink structure created by genrules and its dependency checking of directories is unsound.

And indeed when I run the build I get this warning trying to build this target:

    WARNING: /cockroach/pkg/ui/distccl/BUILD.bazel:3:8: output 'pkg/ui/distccl/assets' of //pkg/ui/distccl:genassets is a directory; dependency checking of directories is unsound

For *now*, if we have to do this, this is fine and we can check it in so we can unblock other work, but we would want to fix this before we can call this production-ready. A couple options:

1. You can declare all the sub-files under the `assets` directory explicitly. I see in that directory there are a lot of files with long unpredictable names (like `02290aae2a70e2d8dc8d391049661b25.woff2`), so maybe this isn't possible? 
2. You can compress the `assets` directory into a single file (like a `.tar.gz`), and then in the Go code you can decompress the `.tar.gz` and expose those artifacts as an `FS`. [Very small example.](https://stackoverflow.com/questions/51073133/bazel-genrule-that-outputs-a-directory)

(2) is probably what you'll have to do, although it'll take a little refactoring on the Go side. Like I said, we can take that as a TODO for later.

Reply:
Oh, there might be a low-friction third option that allows us to accomplish the same goal without a bunch of refactoring. Not really sure, I'd have to look closer. Happy to have a call about this if you want to.